gqcpy/include/interfaces.hpp

```diff
@@ -472,7 +472,21 @@ void bindSimpleTransformationInterface(Class& py_class) {
 
         .def("matrix",
              &Type::matrix,
-             "Return the transformation matrix that collects the expansion coefficients of the new basis (vectors) in the old basis as columns.");
+             "Return the transformation matrix that collects the expansion coefficients of the new basis (vectors) in the old basis as columns.")
+
+        .def("inverse",
+             &Type::inverse,
+             "Return the inverse transformation of this transformation matrix.")
+
+        .def(
+            "isUnitary",
```

Comment:
I've added this binding since I might possible be using this for telling my story in a notebook.